jar-persistence/src/test/java/com/sixsq/slipstream/persistence/VmTest.java

```diff
@@ -225,9 +225,9 @@ public void cloudInstanceIdUserMustBeUnique() throws Exception {
 			EntityTransaction transaction = em.getTransaction();
 			transaction.begin();
 			Query query1 = em
-					.createNativeQuery("insert into Vm values (10, 'lokal', 'instance100', null, null, 'up', 'joe')");
+					.createNativeQuery("insert into Vm values (10, 'lokal', 'instance100', null, null, 'up', 'user')");
 			Query query2 = em
```

Comment:
@st: I think it's better to use the variable user (line 46 of this file) so if at some point the name is changed, it will not break the test.